packages/docs/src/routes/docs/(qwikcity)/middleware/index.mdx

```diff
@@ -51,6 +52,14 @@ export const onDelete: RequestHandler = async (requestEvent) => { ... }
 
 Each middleware function is passed a [`RequestEvent`](#requestevent) object which allows the middleware to control the response.
 
+If you want to combine multiple `RequestHandler`s together, there is a utility function called `combineRequestHandlers()` that will allow you to chain mulitple `RequestHandler`s together.
+```typescript
+import type { RequestHandler } from '@builder.io/qwik-city';
+import { combineRequestHandlers } from "@builder.io/qwik-city";
+
+export const onRequest: RequestHandler = combineRequestHandlers(handler1, handler2, handler3);
+```
+They will be called in the order they are given, so `handler2` can depend on something `handler1` has placed into the `sharedMap`, for example.
 
```

Comment:
I read your PR description and the use case is clear to me.
Can you add that description here too pls?

Reply:
Done.

Reply:
Thanks